### ISSUE TITLE
[BUG] Status and navigation not really working

### ISSUE THREAD

**Issue by Aiuraa** [MM2-0/Kvaesitso#290]
This behaviour causes hide statusbar & navigation feature not working when opening apps and then closes the app, can be described with this video.

https://user-images.githubusercontent.com/104730381/225477164-bfb8d957-bc8c-4d40-b0d1-c6620dbb39ba.mp4

I'm currently at android 10, oh and the monkey patch for this issue is, killing the app makes it working again but only for brief moment before you open any apps again and closes it.

Hopefully this issue can be solved.

**Comment by exstntlmsnthrp** [comment]
I do not understand the problem you are describing or are attempting to demonstrate with the video. Can you try to further explain the problem, please?

**Comment by exstntlmsnthrp** [comment]
Ok, after thinking about it a little further, maybe I understand. It may depend on your settings. Do you have the following settings enabled or disabled?

Appearance > Hide status bar
Appearance > Hide navigation bar

See 👇:

![Screenshot_20230316-162839_Trebuchet.png](https://user-images.githubusercontent.com/57594290/225745607-3a42adaa-f7ce-4901-9a53-5a46ef8de177.png)

**Comment by Aiuraa** [comment]
> Ok, after thinking about it a little further, maybe I understand. It may depend on your settings. Do you have the following settings enabled or disabled?
> 
> Appearance > Hide status bar Appearance > Hide navigation bar
> 
> See 👇:
> 
> ![Screenshot_20230316-162839_Trebuchet.png](https://user-images.githubusercontent.com/57594290/225745607-3a42adaa-f7ce-4901-9a53-5a46ef8de177.png)

yes i do, that's why im reporting the bug, toggling off and on again seem doesn't work too unless i force close the app and it will work again (only for a moment) before you open any apps and then exit that app (as the video describes)

**Comment by exstntlmsnthrp** [comment]
>
> > ![
> 
> oh sorry about late reply since i kinda busy 😅

No problem! Happens to everyone. 

I wonder what's happening? Anything in the logs? 

Maybe @MM2-0 will respond? I'm just another user trying to help out. I'm not able to duplicate the problem with my phone.

**Comment by MM2-0** [comment]
I can't reproduce this on Android 13 but I haven't tried on Android 10 yet.

**Comment by MM2-0** [comment]
Confirmed on Android 10.
